File: Universal Windows Platform Solution/Phantom of the West/Phantom of the West/Visual Novel/Chapters/Chapter 3/CaitlynMightBeGoodMother.cs
using Phantom_of_the_West.Visual_Novel.Serialization.Event_Flagging;

namespace Phantom_of_the_West.Visual_Novel.Chapters.Chapter_3
{
	internal class CaitlynMightBeGoodMother : StoryChoice
	{
		internal CaitlynMightBeGoodMother() : base(StoryChoice.GetLocalizedString("ch3CaitlynMightBeGoodMother"), 3072)
		{
		}

		public override void Select()
		{
			PotWVN vn = PotWVN.MainVN;
			vn.EventFlags.Ch3TalkedWithCaitlyn = TalkWithCaitlyn.SaidMaybeGoodMother;
			base.Select();
		}
	}
}

File: Android Project/app/src/main/java/com/christopherraleigh/phantomofthewest/visual_novel/chapters/chapter_3/CaitlynMightBeGoodMother.java
package com.christopherraleigh.phantomofthewest.visual_novel.chapters.chapter_3;

import com.christopherraleigh.phantomofthewest.R;
import com.christopherraleigh.phantomofthewest.visual_novel.PotWVN;
import com.christopherraleigh.phantomofthewest.visual_novel.chapters.StoryChoice;
import com.christopherraleigh.phantomofthewest.visual_novel.serialization.event_flagging.TalkWithCaitlyn;

/**
 * Created by christopherraleigh on 2016-05-12.
 */
public class CaitlynMightBeGoodMother extends StoryChoice {
    public CaitlynMightBeGoodMother() {
        super(R.string.ch3CaitlynMightBeGoodMother, 3072);
    }

    @Override
    public void select()
    {
        PotWVN vn = PotWVN.getMainVN();
        vn.getEventFlags().setCh3TalkedWithCaitlyn(TalkWithCaitlyn.SaidMaybeGoodMother);
        super.select();
    }
}
